### DISCUSSION TITLE
Sonoff ZigBee 3.0 Donges port failure why?

### DISCUSSION THREAD

**Discussion by ionultd** [Koenkk/zigbee2mqtt#11822] [General]
Hi all

I have tried finding a solution on the www and here but not seem to find anything directly relating to this issue.

I have installed Zigbee2Mqtt on UnRaid
I have connected the Sonoff Zigbee 3.0 Dongle with firmware CC1352P2_CC2652P_launchpad_coordinator_20211217.hex on it
Config File is:
`homeassistant: true
permit_join: false
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://xxxx
  user: mailbox
  password: xxxxx
serial:
  adapter: ezsp
  port: >-
    /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_f008d71dbac9eb1185b68d4f1d69213e-if00-port0
frontend:
  port: 9442
advanced:
  network_key:
    - 167
    - 45
    - 183
    - 39
    - 214
    - 48
    - 124
    - 207
    - 1
    - 112
    - 49
    - 154
    - 178
    - 27
    - 148
    - 65`
I keep getting this error
Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/serial/by-id/usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_f008d71dbac9eb1185b68d4f1d69213e-if00-port0'
ls -l /dev/serial/by-id
lrwxrwxrwx 1 root root 13 Mar 12 13:41 usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_f008d71dbac9eb1185b68d4f1d69213e-if00-port0 -> ../../ttyUSB0

I tried using the port designation but it gives the same error 
Error: Error: No such file or directory, cannot open  /dev/ttyUSB0
at SerialPort.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:201:28)

What am i doing wrong?

**Comment by DesertF0x** [comment]
The device is only accessible for root, what user is zigbeemqtt running at?

**Reply by ionultd** [reply]
@DesertF0x thx for replying. It is installed by UnRaid so I would assume it has the correct permissions but as I cannot start the instance due to the error I cannot check on the user permissions. Thing is that I have no problem running it with an older cc2531 Zigbee Dongle

**Reply by DesertF0x** [reply]
Can you check the permissions of that dongle? Is it root root too or is it root dailout?

**Reply by ionultd** [reply]
@DesertF0x Pray tell how do I do that :-)

**Reply by DesertF0x** [reply]
Ls -la /dev/ttyXXXX on the dev of the usb stick like in your post above.

**Reply by ionultd** [reply]
crw-rw---- 1 root dialout 188, 0 Mar 13 17:50 /dev/ttyUSB0

**Reply by DesertF0x** [reply]
I have both sticks connected to my system, too:
```
ls -la /dev/serial/by-id/
insgesamt 0
drwxr-xr-x 2 root root 80 Mär 11 14:28 .
drwxr-xr-x 4 root root 80 Mär  8 15:25 ..
lrwxrwxrwx 1 root root 13 Mär 15 06:52 usb-ITead_Sonoff_Zigbee_3.0_USB_Dongle_Plus_e6b0598c8fc9eb11b873914f1d69213e-if00-port0 -> ../../ttyUSB0
lrwxrwxrwx 1 root root 13 Mär 15 06:52 usb-Texas_Instruments_TI_CC2531_USB_CDC___0X00124B0014D96950-if00 -> ../../ttyACM0

ls -la /dev/ttyUSB0 
crw-rw---- 1 root dialout 188, 0 Mär 15 06:52 /dev/ttyUSB0

ls -la /dev/ttyACM0 
crw-rw---- 1 root dialout 166, 0 Mär 15 06:52 /dev/ttyACM0
```
can you check if the permissions are the same on your side?

**Reply by DesertF0x** [reply]
Are you running zigbee2mqtt in a container?

**Reply by ionultd** [reply]
My permissions are exactly the same as yours.  UnRaid does use containers, so yes my Zigbee2MQTT is in a container.

**Reply by ionultd** [reply]
@DesertF0x Do you have any further advice?

**Reply by DesertF0x** [reply]
If the dev name is different like on my system have you added the right device to the docker container then?

**Reply by ionultd** [reply]
@DesertF0x Well I do not believe the dev name is different as my previous working cc2531 port was this: /dev/serial/by-id/usb-1a86_USB_Serial-if00-port0

**Reply by DesertF0x** [reply]
In my System it is different. Anyway check if you have forwarded the right device to the docker container.

**Comment by rbrowning85** [comment]
Did this get resolved? I am having the same issue.

**Reply by DesertF0x** [reply]
This was most probably a config error. Have you checked the above mentioned things?
